scripts/easy_train.py

```diff
@@ -1433,74 +1446,90 @@ def str2bool(v):
-    parser.add_argument("--threads", default=-1, type=int, dest='threads', help="Number of torch threads to use. Default automatic (cores) .")
+    parser.add_argument("--num-workers", default=default_data_loader_threads, type=int, dest='num_workers', help="Number of worker threads to use for data loading (binpacks). Increase with large skipping rates, or underloaded GPUs")
+    parser.add_argument("--batch-size", default=16384, type=int, dest='batch_size', help="Number of positions per batch / per iteration.")
+    parser.add_argument("--threads", default=default_pytorch_threads, type=int, dest='threads', help="Number of torch threads to use.")
     parser.add_argument("--seed", default=42, type=int, dest='seed', help="torch seed to use.")
     parser.add_argument("--smart-fen-skipping", default=True, type=str2bool, dest='smart_fen_skipping', help="If used then no smart fen skipping will be done. By default smart fen skipping is done.")
     parser.add_argument("--wld-fen-skipping", default=True, type=str2bool, dest='wld_fen_skipping', help="If used then no wld fen skipping will be done. By default wld fen skipping is done.")
     parser.add_argument("--random-fen-skipping", default=3, type=int, dest='random_fen_skipping', help="skip fens randomly on average random_fen_skipping before using one.")
-    parser.add_argument("--start-from-model", default=None, type=str, dest='start_from_model', help="Initializes training using the weights from the given .pt model")
+    parser.add_argument("--start-from-model", default=None, type=str, dest='start_from_model', help="Initializes training using the weights from the given .pt .ckpt .nnue model")
     parser.add_argument("--start-from-experiment", default=None, type=str, dest='start_from_experiment', help="Initializes training using the best network from a given experiment (by name). Uses best net from ordo, fallbacks to last.")
-    parser.add_argument("--gpus", type=str, dest='gpus', default='0')
-    parser.add_argument("--runs-per-gpu", type=int, dest='runs_per_gpu', default=1)
-    parser.add_argument("--features", type=str, help="The feature set to use")
+    parser.add_argument("--start-from-stockfish-net", default=False, type=str2bool, dest='start_from_stockfish_net', help="Initializes training using the weights from the .nnue associated with --engine-test-branch")
+    parser.add_argument("--gpus", type=str, dest='gpus', default='0', help="a GPU ID or a list of GPU IDs.")
+    parser.add_argument("--runs-per-gpu", type=int, dest='runs_per_gpu', default=1, help="To increase the load on strong GPUs run more than one repetition of this experiment and reduce net variance.")
+    parser.add_argument("--features", type=str, default='HalfKAv2_hm^', help="The feature set to use") # TODO can this be made a default based on the nnue-pytorch-branch specified?
```

Comment:
yea, possibly by searching for `_default_feature_set_name = 'HalfKAv2_hm^'` in features.py. Needs to happen after nnue-pytorch setup

Reply:
I'll pass on this one, as I don't know this well enough.